collections/Auct_F/MS_Auct_F_2_8.xml

```diff
@@ -37,49 +36,328 @@
                      <idno type="SCN">8851</idno>
                   </altIdentifier>
                </msIdentifier>
+               <head>The <q>Oxford Virgil</q></head>
```

Comment:
perhaps better to have something more descriptive? e.g. Works of Virgil with glosses ('The Oxford Virgil'); France (Saint-Germain-des-Prés), 9th century, first half